service/controller/app/v1/resource/chartcrd/create.go

```diff
@@ -0,0 +1,59 @@
+package chartcrd
```

Comment:
Why does it need to create this CRD? Isn't chart-operator running in the tenant cluster?

Reply:
I had the same question in https://github.com/giantswarm/app-operator/pull/33#discussion_r254598559. I would like to understand this better as well.

Reply:
In the case of chart-operator was not running in tenant cluster beforehand, app-operator could not create `chart` CR. That's what I got from @rossf7, and he pointed out that another place that same practice is ongoing, https://github.com/giantswarm/cluster-operator/pull/377/files#diff-6ec70deab7af8061be1a5eb8b75ed5e9R20
WDYT?

Reply:
It would be good to have a diagram to visualize who is supposed to do what. I do not understand the circumstances in which who should install what.

Reply:
I would suggest to make it another PR in here, https://github.com/giantswarm/sig-operator, 
Just a min.

Reply:
Not sure. This here is pretty specific and not a general thing as I understood it. Maybe just try to explain how this here is supposed to look like, as it does not seem to be clear for several people now.

Reply:
Does chart-operator run in the tenant cluster? I think yes, because how otherwise it would watch CRs from multiple clusters. If so why app-operator can't cancel for as long as chart CRD isn't created by the chart-operator?